core/src/main/scala/org/apache/spark/api/java/JavaRDD.scala

```diff
@@ -103,7 +103,8 @@ class JavaRDD[T](val rdd: RDD[T])(implicit val classTag: ClassTag[T])
    * @param withReplacement can elements be sampled multiple times (replaced when sampled out)
    * @param fraction expected size of the sample as a fraction of this RDD's size
    *  without replacement: probability that each element is chosen; fraction must be [0, 1]
-   *  with replacement: expected number of times each element is chosen; fraction must be &gt;= 0
+   *  with replacement: expected number of times each element is chosen; fraction must be greater
+   *  than or equal to 0
```

Comment:
Oh, yes. Exactly. It was not an error but it was printed as they are (`&gt;` as `&amp;gt;`). I had a hard time to figure this out in https://github.com/apache/spark/pull/16013#discussion_r89666914

I haven't looked into this super deeper as the output is incorrect anyway but I suspect this [replacement](https://github.com/typesafehub/genjavadoc/blob/861f3fdf78fb98f8f3d879f7734efc8d78b3b983/plugin/src/main/scala/com/typesafe/genjavadoc/BaseComments.scala#L25) in genjavadoc.